docs/.vuepress/config.js

```diff
@@ -296,17 +276,12 @@ module.exports = {
             '/project/repository-guide',
             '/project/related-projects',
             '/project/implementation-status',
-            ['https://github.com/ipfs/roadmap', 'Roadmap'],
```

Comment:
Why are we removing the link to the roadmap?

Reply:
It's not clear to me that this repo is still being used. Last commit was end of October. Is this repo no longer being used for roadmap stuff? Aren't roadmaps individualized to a product now?